### DISCUSSION TITLE
Recently switched boards on my Ender 6

### DISCUSSION THREAD

**Discussion by RGBMainframe** [Desuuuu/DGUS-reloaded-Klipper#20] [Q&A]
One of the drivers on my Creality 4.3.1 board went bad and swapped over to an SKR Mini E3 v2. Previously had it working great with the stock board, but now I can't seem to get it to work. I'm guessing it's using different pins? I've tried all available options USART1/2/3 to no avail. 

Any guidance on getting this working with a main board other than stock?

**Comment by Desuuuu** [comment]
STM32 is supported so this shouldn't be a problem. Follow these steps:

* Look up your board's documentation to figure out which USART interface map to which physical port
* Make sure you wire the screen properly to one of the available ports (RX<->TX, TX<->RX)
* Configure the microcontroller firmware to use the interface that's connected to that port and flash it

The configuration for the screen on the host shouldn't require modifications.
